java/clients/src/test/java/sleeper/clients/QueryWebSocketCommandLineClientTest.java

```diff
@@ -477,11 +489,14 @@ private Query rangeQuery(String queryId, long min, long max) {
     }
 
     protected void runQueryClient(String queryId, Client webSocketClient) throws Exception {
-        new QueryWebSocketCommandLineClient(instanceProperties, tableIndex, new FixedTablePropertiesProvider(tableProperties),
-                in.consoleIn(), out.consoleOut(), new QueryWebSocketClient(instanceProperties,
-                        new FixedTablePropertiesProvider(tableProperties), out.consoleOut(), webSocketClient),
-                () -> queryId)
-                .run();
+        try {
+            new QueryWebSocketCommandLineClient(instanceProperties, tableIndex, new FixedTablePropertiesProvider(tableProperties),
+                    in.consoleIn(), out.consoleOut(), new QueryWebSocketClient(instanceProperties,
+                            new FixedTablePropertiesProvider(tableProperties), out.consoleOut(), webSocketClient),
+                    () -> queryId)
+                    .run();
+        } catch (CompletionException e) {
+        }
```

Comment:
Actually i'm not even sure we want exceptions to be thrown out into the command line client. It means you have to run the client again if you want to run a query again, whereas if we just output to the console that an exception happened with the message, then it can gracefully continue.

Reply:
I've caught the CompletionException in the client code and removed the try/catch in the test